packages/shared-components/src/components/onto-header/onto-header.tsx

```diff
@@ -1,18 +1,39 @@
 import { Component, Host, h } from '@stencil/core';
+import {WorkbenchServiceProvider, WorkbenchEventService, WorkbenchEventType} from "@ontotext/workbench-api";
 
 @Component({
   tag: 'onto-header',
   styleUrl: 'onto-header.scss',
   shadow: false,
 })
 export class OntoHeader {
+
+  private eventService: WorkbenchEventService;
+
+  private locale = 'en'
```

Comment:
So, in how many places we need this default to be declared?!

Reply:
This is code just for testing. It will be removed in next request when language selector is implemented.

Reply:
So, put a TODO maybe, because I didn't think of this :)

Reply:
Added comment